web/src/services/editorStateService.ts

```diff
@@ -17,6 +17,14 @@ const editorStateService = {
   setMemoVisibility: (memoVisibility: Visibility) => {
     store.dispatch(setMemoVisibility(memoVisibility));
   },
+
+  setResourceList: (resourceList: Resource[]) => {
+    store.dispatch(setResourceList(resourceList));
```

Comment:
Needs to check duplicated resource. I can select the same resource again when editing resource.

<img width="402" alt="image" src="https://user-images.githubusercontent.com/24653555/204283967-afe07d7c-7108-4b76-b756-3fd2ed612b2e.png">

Reply:
When we set the edit memo, we need to update the resourceList state together.

Reply:
When we set the edit memo, we need to update the resourceList state together.